docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
```

Comment:
Is there a reason to not use the version 3?

Reply:
@mcarmonaa - there is no any reason! I just sticked with *v2*  for a long time, so kept it here for backwards compatibility, that's it. Frankly, I'm totally open to upgrade it to any version.